Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mono:latest
```

Comment:
4.051,
On Jun 20, 2016 8:32 PM, "Michael" <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/QuantConnect/Lean/pull/440#discussion_r67791150>:
>
> > @@ -0,0 +1,18 @@
> > +FROM mono:latest
>
> I wish I could remember, I think 4.1, but am not positive.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/QuantConnect/Lean/pull/440/files/d088f8ca8ffade77d6ca5fe433c2e78d84201e28#r67791150>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACI6mS8C73oynBe1ZC1ELwPyphSU__HHks5qNzEjgaJpZM4I6Mdh>
> .
>